libraries/chain/block_header_state_legacy.cpp

```diff
@@ -210,9 +210,8 @@ namespace eosio::chain {
       if (new_finalizer_policy) {
          new_finalizer_policy->generation = 1;
          // set current block_num as qc_claim.last_qc_block_num in the IF extension
-         qc_claim_t initial_if_claim { .last_qc_block_num = block_num,
-                                       .last_qc_block_timestamp = timestamp,
-                                       .is_last_qc_strong = false };
+         qc_claim initial_if_claim { .block_num = block_num,
```

Comment:
Was trying to be consistent with other types in finality_core.hpp which do not have `_t`. Was struggling with naming the qc claim in the `instant_finality_extension`. Do you and @heifner have any better name for that field?

Reply:
It would be better to add `_t`  to the types in `finality_core.hpp`  rather than struggling to find variable or members name when `qc_claim` is ideal.

Reply:
As a note, `finality_core.hpp` defines `block_num_type` and `block_time_type`, so it is altready inconsistent with the naming convention for types.
I propose using `_t` for all types, like the STL does for example in `uint32_t` or `std::invoke_result_t,`.

Reply:
I'll change to qc_claim_t

Reply:
Done.